defender/downloaded/Forta_Large_Borrows_Governance.js

```diff
@@ -187,13 +187,9 @@ exports.handler = async function (autotaskEvent) {
   // wait for the promises to settle
   let results = await Promise.allSettled(promises);
 
-  // construct the Etherscan transaction link
-  const etherscanLink = `[TX](<https://etherscan.io/tx/${transactionHash}>)`;
-
   // create promises for posting messages to Discord webhook
   const discordPromises = results.map((result) => {
-    console.log(`${etherscanLink} ${result.value}`);
-    return postToDiscord(discordUrl, `${etherscanLink} ${result.value}`);
+    return postToDiscord(discordUrl, `${result.value}`);
```

Comment:
The deleted `console.log()` statement is how we were forwarding information to Dune.
```suggestion
    // with Log Forwarding enabled, this console.log will forward the text string to Dune Analytics
    console.log(`${result.value}`);
    return postToDiscord(discordUrl, `${result.value}`);
```